automated install/basic-install.sh

```diff
@@ -855,7 +855,7 @@ setDHCPCD() {
         echo "interface ${PIHOLE_INTERFACE}
         static ip_address=${IPV4_ADDRESS}
         static routers=${IPv4gw}
-        static domain_name_servers=127.0.0.1" | tee -a /etc/dhcpcd.conf >/dev/null
+        static domain_name_servers=${PIHOLE_DNS_1},${PIHOLE_DNS_2}" | tee -a /etc/dhcpcd.conf >/dev/null
```

Comment:
@PromoFaux
It needs to be a space separate list: https://wiki.archlinux.org/index.php/Dhcpcd#Static_profile (sorry Debian man page did not contain this info/example with multiple IPs)

Reply:
OK, I hadn't actually tested it. So, like so?

```suggestion
        static domain_name_servers=${PIHOLE_DNS_1} ${PIHOLE_DNS_2}" | tee -a /etc/dhcpcd.conf >/dev/null
```

Reply:
@PromoFaux 
Jep that should be it, will also run some test.